deployments/helm/eosxd-csi/values.yaml

```diff
@@ -212,7 +231,9 @@ nodeplugin:
     resources: {}
     # Extra volume mounts to append to nodeplugin's
     # Pod.spec.containers[name="mountreconciler"].volumeMounts.
-    extraVolumeMounts: []
+    extraVolumeMounts:
+      - name: eos-csi-dir-etc-krb5-conf
+        mountPath: /etc/krb5.conf.d
```

Comment:
Do we need a subpath or the file will be just be  `crypto-policies`?

Reply:
I didnt need a subpath as all the entries in the config map are written as a file under `/etc/krb5.conf.d` so this resolves to the original path of `/etc/krb5.conf.d/crypto-policies`

Reply:
I just followed the same as what Robert set up in the automount plugins volume mounts

Reply:
In this case I don't think it matters as my configmap only has one file inside of it anyway